itenium-socks/src/app/app.component.ts

```diff
@@ -1,5 +1,7 @@
-import { Component } from '@angular/core';
-import { RouterOutlet } from '@angular/router';
+import {Component} from '@angular/core';
+import {RouterOutlet} from '@angular/router';
+import {registerLocaleData} from "@angular/common";
```

Comment:
Avoid useless changes!
Respect the coding style of the project

Doing this is basically raping your git history

+ The next developer creates the following PR which adds the spaces all over again 
etc etc etc